src/js/components/Video/stories/Simple.js

```diff
@@ -8,22 +8,7 @@ const SimpleVideo = props => (
   <Grommet theme={grommet}>
     <Box align="center" pad="large">
       <Video {...props}>
-        <source
-          src="http://techslides.com/demos/sample-videos/small.webm"
-          type="video/webm"
-        />
-        <source
-          src="http://techslides.com/demos/sample-videos/small.ogv"
-          type="video/ogg"
-        />
-        <source
-          src="http://techslides.com/demos/sample-videos/small.mp4"
-          type="video/mp4"
-        />
-        <source
-          src="http://techslides.com/demos/sample-videos/small.3gp"
-          type="video/3gp"
-        />
+        <source src="small.mp4" type="video/mp4" />
```

Comment:
I'm wondering if we should keep the other source tags since it's optional anyway and it will demonstrate better how the tag could be used.
```
The <source> element allows you to specify alternative video files which the browser may choose from. The browser will use the first recognized format.
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source

Reply:
Yeah I agree, I will add it in the next commit